apps/style/javaIde/style.scss

```diff
@@ -0,0 +1,16 @@
+.container {
+  width: 95%;
+}
+
+body {
+  background-color: #1b1c17 !important;
+  color: white;
+} 
+
+.main {
+  padding: 0 !important;
```

Comment:
Is it possible to also remove these other two `!important`s?

Reply:
interesting...I swear I needed them when I first wrote this but now I don't. I'll remove and keep an eye out if we see issues.